test/parallel/test-fs-long-path.js

```diff
@@ -4,6 +4,11 @@ var fs = require('fs');
 var path = require('path');
 var assert = require('assert');
 
+if (!common.isWindows) {
+  console.log('1..0 # Skipped: this test is Windows-specific.');
+  process.exit();
```

Comment:
Can you use `return` here?  That's a little more idiomatic, that's what other tests do.  LGTM apart from that.